tests/integration_python/pixi_build/test_build.py

```diff
@@ -313,3 +313,47 @@ def test_smokey(pixi: Path, build_data: Path, tmp_pixi_workspace: Path) -> None:
     metadata = json.loads(conda_meta.read_text())
 
     assert metadata["name"] == "smokey"
+
+
+@pytest.mark.slow
+def test_recursive_source_run_dependencies(
+    pixi: Path, build_data: Path, tmp_pixi_workspace: Path
+) -> None:
+    """
+    Test whether recursive source dependencies work properly if
+    they are specified in the `run-dependencies` section
+    """
+    project = "recursive_source_run_dep"
+    test_data = build_data.joinpath(project)
+
+    shutil.copytree(test_data, tmp_pixi_workspace, dirs_exist_ok=True)
+    manifest_path = tmp_pixi_workspace.joinpath("pixi.toml")
+
+    # TODO: Setting the cache dir shouldn't be necessary!
```

Comment:
Could you please add a small comment on why it's not necessary? it would be easier to get the idea without knowing the full context